Tools/ApplicationDebugger/jacobi/sample.json

```diff
@@ -0,0 +1,174 @@
+{
+  "guid": "CFAC8EF8-E362-4054-ABE3-A04EEBEBCB46",
+  "name": "Debugger: Jacobi",
+  "categories": ["Toolkit/Get Started", "Toolkit/oneAPI Tools/Application Debugger"],
```

Comment:
We added "Debugger" to array transform to ensure it was ordered after vector add in the sample browser/IDE.  we don't want this in the Get Started category either.  so, remove name should be: "name": "Jacobi", and category:   "categories": ["Toolkit/oneAPI Tools/Application Debugger"],